src/lib/es5.d.ts

```diff
@@ -1669,13 +1669,17 @@ interface DataView {
      * Gets the Float32 value at the specified byte offset from the start of the view. There is
      * no alignment constraint; multi-byte values may be fetched from any offset.
      * @param byteOffset The place in the buffer at which the value should be retrieved.
+     * @param littleEndian If false or undefined, a big-endian value should be written,
+     * otherwise a little-endian value should be written.

```

Comment:
yes.

I just copied the parameter descriptions related to `setXXX` before, without noticing the subtle difference in semantics, `set` corresponds to `write`, and `get` corresponds to `read`. So I checked all the methods and made corresponding changes.

Other than that, I agree with you on the 'otherwise' part. Because the parameter name `littleEndian` itself can explain its meaning, adding this sentence seems a bit redundant. Similarly, in addition to the `getXXX` methods, I have also removed 'otherwise' from the other `setXXX` methods, which helps to keep the documentation consistent.